flink-end-to-end-tests/test-scripts/common_s3.sh

```diff
@@ -72,4 +75,22 @@ function s3_setup_with_provider {
   set_config_key "$2" "com.amazonaws.auth.EnvironmentVariableCredentialsProvider"
 }
 
-source "$(dirname "$0")"/common_s3_operations.sh
\ No newline at end of file
+###################################
+# Fetches the result in part files into the OUTPUT_PATH.
+#
+# Globals:
+#   OUTPUT_PATH, OUT
+# Arguments:
+#   $1 - single?
```

Comment:
Let's expand this a _bit_, like "true if result is a single file, false if result is a collection of part- files"